src/simulation/dynamics/reactionWheels/reactionWheelStateEffector.cpp

```diff
@@ -23,6 +23,8 @@
 #include <cstring>
 #include <iostream>
 #include <cmath>
+#include "architecture/messaging/messaging.h"
```

Comment:
These includes are already in the `reactionWheelStateEffector.h` file and don't need to be added here.